### PR TITLE
added gradients to dark dashboard and dark booking page

### LINKED ISSUE #8458: [CAL-1581] experiment: bg-gradient on dark mode
image

image

<sub>[CAL-1581](https://linear.app/calcom/issue/CAL-1581/experiment-bg-gradient-on-dark-mode)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
CleanShot 2023-04-22 at 17 30 57@2x

### CODE REVIEW

File: apps/web/components/booking/AvailableTimes.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
   return (
     <div ref={slotPickerRef}>
       {!!date ? (
-        <div className="mt-8 flex h-full w-full flex-col rounded-md px-4 text-center sm:mt-0 sm:p-4 md:-mb-5 md:min-w-[200px] md:p-4 lg:min-w-[300px]">
+        <div className="mt-8 flex h-full w-full flex-col rounded-md px-4 text-center dark:bg-gradient-to-tr dark:from-[#1c1c1c] dark:to-[#2a2a2a] sm:mt-0 sm:p-4 md:-mb-5 md:min-w-[200px] md:p-4 lg:min-w-[300px]">
```
Comment (PeerRich) [inline apps/web/components/booking/AvailableTimes.tsx:81]: we may wanna create tokens here @sean-brydon if we do want to merge this.

will wait for some results https://twitter.com/peer_rich/status/1649798977003323392

**Comment by PeerRich** [comment]
https://twitter.com/peer_rich/status/1649832406055092224

**Comment by PeerRich** [comment]
probably not worth merging here, instead waiting for @JeroenReumkens to merge his new booker and then consider adding it there

**Review by Udit-takkar — APPROVED** [review]
Screenshot 2023-05-08 at 1 54 20 PM

LGTM

**Review by JeroenReumkens — APPROVED** [review]
LGTM, love the gradients!

**Pull request merged**